CHANGES.md

```diff
@@ -1,3 +1,9 @@
+1.1.0
+* Added SignConfiguration POST API
```

Comment:
I would simplified description to something like: _Added SignConfiguration management operations_.  It doesn't bring much value for customers information that we are handling all http operations ;)

Reply:
Updated the description.